packages/single-invoice/src/utils/getConversionPaymentValues.d.ts

```
@@ -0,0 +1,42 @@
+/**
+ * Utility method to compute various settings associated to a payment involving conversion only
+ */
+export declare const getConversionPaymentValues: ({ baseAmount, denominationCurrency, selectedPaymentCurrency, currencyManager, provider, fromAddress, }: {
+    baseAmount: number;
+    denominationCurrency: CurrencyTypes.CurrencyDefinition;
+    selectedPaymentCurrency: CurrencyTypes.CurrencyDefinition;
+    currencyManager: CurrencyManager;
+    provider?: providers.Provider;
+    fromAddress?: string;
+}) => Promise<ConversionPaymentValues | undefined>;
+export {};
```

Biome (1.9.4)

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)